_includes/sass/variables/_colors-v2.scss

```diff
@@ -6,8 +6,14 @@ $orange:                                         #ff5722;
 $yellow:                                         #ffcc00; // v1
 $green:                                          #3a8717;
 $blue:                                           #00bbbb; // v1
+$light-blue:                                     #d4f2f2;
+$lighter-blue:                                   #edfafa;
+$dark-blue:                                      #009595;
+$dark-aqua:                                      #007b7b;
```

Comment:
I'm wondering if we should have values for these colors in our other color palettes as well, or have some default. Almost like having implementations of a color palette interface. That way we there are no assumptions needed when using a color variables. What do you think?